src/ros2_gamepad_to_twist_message/ros2_gamepad_to_twist_message/playstation.py

```diff
@@ -44,8 +44,10 @@ def __init__(self, publish_topic='/cmd_vel', publish_frequency=100):
         super().__init__('gamepad_publisher')
 
         # publisher initialization
-        self.publisher_ = self.create_publisher(Twist, publish_topic, 1)
-        self.publisher2 = self.create_publisher(String, '/trigger', 1)
+        self.pub_joystick = self.create_publisher(Twist, publish_topic, 1)
+        self.pub_camera_trigger = self.create_publisher(Bool, '/camera/cam_0/snapshot/trigger', 2)
```

Comment:
Ideally these are changed to variable arguments from the configuration file, but lets not do that now